base/src/com/google/idea/blaze/base/buildview/BazelExecService.kt

```diff
@@ -76,32 +78,6 @@ class BazelExecService(private val project: Project, private val scope: Coroutin
     return process
   }
 
-  private suspend fun awaitProcess(ctx: BlazeContext, hdl: ProcessHandler): Int {
-    hdl.addProcessListener(object : ProcessListener {
-      override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
-        if (outputType == ProcessOutputTypes.SYSTEM) {
-          ctx.println(event.text)
-        } else {
-          ctx.output(PrintOutput.process(event.text))
-        }
-
-        LOG.debug("BAZEL OUTPUT: " + event.text)
```

Comment:
This is kind of required in order to forward any output to the build console.

Reply:
It wasn't used at all with either build or run after removal of `WritingOutputSink` and `PtyCommandLine` is capturing command stdout instead just nice.